templates/garden/nightly/keras-api.libsonnet

```diff
@@ -33,6 +33,9 @@ local tpus = import "../../tpus.libsonnet";
       "-c",
       base_command + " behave -e ipynb_checkpoints --tags=-fails -i %s" % self.testFeature,
     ],
+    regressionTestConfig: {
+      alert_for_failed_jobs: false,
```

Comment:
@will-cromar `false` or `False`? We use lowercase [here](https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/templates/base.libsonnet#L41) but that could also be wrong

Reply:
Lower-case is correct https://jsonnet.org/ref/spec.html